src/fillTank.test.js

```diff
@@ -1,11 +1,169 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('is declared', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('does nothing when amount < 2', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 40, 1.95);
+
+    expect(customer).toEqual({
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    });
+  });
+
+  it('full tank if amount not given', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 40);
```

Comment:
The test case assumes a specific price per unit of fuel, but this price is not provided in the function call. Ensure that the `fillTank` function can handle this scenario correctly or adjust the test to include a price.